storm-server/pom.xml

```diff
@@ -60,6 +60,7 @@
         <dependency>
             <groupId>org.rocksdb</groupId>
             <artifactId>rocksdbjni</artifactId>
+            <version>${rocksdb-version}</version>
```

Comment:
just curious why this was needed

Reply:
Not specifying a version in the dependency can cause unexpected and changing behavior. Maven will use the latest version of the dependency from local/remote repository. Storm pom files have this issue in a lot of places.

Reply:
The version is not specified - only a maven variable is defined. Defining the variable rocksdb-version and not using it, defeats the purpose.

Reply:
I missed that. Since the main pom has dependency management we don't need version here. Will remove this.